### DISCUSSION TITLE
Discussion: Version Release Dates

### DISCUSSION THREAD

**Discussion by zhangchi1** [fivetran/airflow-provider-great-expectations#72] [General]
Hi @denimalpaca @kaxil, thanks again for approving the PR and maintaining this operator. We are actively spiking and trying to onboard great_expectations airflow operator into our DQ system. And we are happy to keep contributing to this fantastic library. We are wondering do we have any rules for the new version release? For example, how often do we release a new version/tag? How are we managing the version number aligned with the great_expectation python library? 

Thanks,

**Comment by kaxil** [comment]
Hi @zhangchi1 Currently the releases are on an ad-hoc basis. For example, if we fix some bugs, we will cut a new version sooner.

@denimalpaca Can tell us more about version alignment with the great_expectation python library

**Comment by denimalpaca** [comment]
Going to pass the buck to @talagluck about version alignment with the `great_expectations` python library, as I don't know how it was currently managed. Earlier this year we did a big overhaul based on the new API version, otherwise I don't have details about maintenance since.

**Comment by talagluck** [comment]
Hi @kaxil - there should generally not be any major issues with version alignment, since we have a lot of notice for more major breaking changes. For bugs and regressions, we are in the process of implementing more formal testing now, which will consist of cloning the Operator repo in our CI/CD and running the tests against the latest version of Great Expectations.
